go/genesis/api/api_test.go

```diff
@@ -136,9 +136,9 @@ func TestGenesisSanityCheck(t *testing.T) {
 
 	testRuntimeID := hex2pk("0000000000000000000000000000000000000000000000000000000000000001")
 	testRuntime := &registry.Runtime{
-		ID:         testRuntimeID,
-		Kind:       registry.KindCompute,
-		KeyManager: testKMRuntime.ID,
+		ID:            testRuntimeID,
+		Kind:          registry.KindCompute,
+		KeyManagerOpt: &testKMRuntime.ID,
```

Comment:
Why the rename to `KeyManagerOpt`? Whether something is optional or required shouldn't be in the name.

Reply:
oh I did that to make sure I would notice all usages. I'll change it back